lib/assets/javascripts/_modules/search.js

```diff
@@ -169,8 +169,9 @@
 
     this.processContent = function processContent (content, query) {
       var output
-      content = '<div>' + content + '</div>'
-      content = $(content).mark(query)
+      $('html').append('<div id="tmp-div-for-sanitising-html"></div>')
+      var sanitizedContent = $('#tmp-div-for-sanitising-html').detach().text(content).html()
```

Comment:
This worked for me, without the need to attach and then detach a node:

```suggestion
      var sanitizedContent = $('<div id="tmp-div-for-sanitising-html"></div>').text(content).html()
```

Reply:
Ahh, nice one!